src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -102,10 +102,13 @@ func NewStartCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 			if len(seeds) == 0 {
 				// If --seeds wasn't passed, fall back to the
 				// env var.
-				seeds = strings.Split(
+				envSeeds := strings.Split(
```

Comment:
also this logic seems odd. 

is it true for all variables that environment comes first? odd that this isn't supported out of the box by viper.

Reply:
This is a fallback. The hierarchy is: flag > env var > config.
viper does support binding to env vars, but for simple types (e.g strings). Ultimately, seed addrs need to go into a `config.SeedServer`, which isn't just a string, but a struct with ID, host and port fields.

Reply:
lgtm